actions/image_partition_action.go

```diff
@@ -1,3 +1,106 @@
+/*
+ImagePartition Action
+
+This action allows to create the disk image and partitions layout mapped to
+root filesystem.
```

Comment:
I don't understand what this means. Something like `This action creates an image file, partitions it and formats the filesystems` would be better

Reply:
ok